app/util/PlutoMessageConsumer.scala

```diff
@@ -0,0 +1,88 @@
+package util
+
+import data.UnpackedDataStores
+import akka.actor.{ActorSystem, Scheduler}
+import org.cvogt.play.json.Jsonx
+import scala.annotation.tailrec
+import scala.concurrent.duration._
+import play.api.libs.json._
+import com.amazonaws.services.sqs.model.{DeleteMessageRequest, ReceiveMessageRequest, Message}
+import scala.concurrent.ExecutionContext
+import com.gu.media.logging.Logging
+import data.DataStores
+import scala.collection.JavaConverters._
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class PlutoMessageConsumer(val stores: DataStores, awsConfig: AWSConfig)
+  extends UnpackedDataStores with Logging {
+
+  val actorSystem = ActorSystem("PlutoMessageConsumer")
+
+
+  def start(scheduler: Scheduler)(implicit ec: ExecutionContext): Unit = {
+    log.info("Starting uploads sqs queue reader")
```

Comment:
Might be worth being clearer as to which sqs we are using, this might be in the `awsConfig`?